### ISSUE TITLE
Extra line breaks on <button> with no text content

### ISSUE THREAD

**Issue by 03juan** [feliperenan/heex_formatter#56]
I had this Bulma #modal-close button, fairly deeply-indented with no text content

```html
            <button class="modal-close is-large" aria-label="close"
                phx-click="cancel-admin"></button>
```

On v0.1 it formatted to

```html
            <button class="modal-close is-large" aria-label="close" phx-click="cancel-admin"></button>
```

And now on master it's adding an extra line break.

```html
            <button class="modal-close is-large" aria-label="close" phx-click="cancel-admin">
              
            </button>
```

Technically all buttons should have [inner text for accessibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#icon_buttons) and Bulma does the right thing by hiding the text through CSS, so maybe this is working as intended, but it's maybe unexpected for people unfamiliar with the spec.

**Comment by 03juan** [comment]
I should add it's not just a line-break, but it also fills the indentation up to the correct place. You'll see it if you select all text in the last code block.
